recipe/meta.yaml

```diff
@@ -1,32 +1,35 @@
-{% set version = "0.7.3" %}
+{% set version = "0.8.0" %}
 
 package:
   name: hvplot
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/h/hvplot/hvplot-{{ version }}.tar.gz
-  sha256: 74b269c6e118dd6f7d2a4039e91f16a193638f4119b4358dc6dbd58a2e71e432
+  sha256: 8630dba34969b105e267cbe14237e2d56d9969c801980ae5e5190bc18b2792d6
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  # ModuleNotFoundError: No module named 'hvplot.__main__'
+  # entry_points:
+  #   - hvplot = hvplot.__main__:main
```

Comment:
We can remove commented `entry_points`, they are empty https://github.com/holoviz/hvplot/blob/04f6dae12c449987abf69601be47c1e292e1ab3a/setup.py#L139

Reply:
Done! Thank you